example/rcnn/rcnn/warmup.py

```diff
@@ -0,0 +1,59 @@
+from mxnet.lr_scheduler import LRScheduler
```

Comment:
@sxjscience how may gpu do you using when training? because the gpu number will influnce the batch-size, so it should be care of --factor-step(=5000 default).  my reproduce is 2gpu, and lr=0.001 in the fisrt 5 epoch, and lr=0.0001 in the last 2 epoch.